pkg/server/status_test.go

```diff
@@ -1478,6 +1478,7 @@ func TestStatusAPICombinedTransactions(t *testing.T) {
 	defer log.Scope(t).Close(t)
 
 	params, _ := tests.CreateTestServerParams()
+	params.Knobs.SpanConfig = &spanconfig.TestingKnobs{ManagerDisableJobCreation: true} // TODO(irfansharif): #74919.
```

Comment:
Not sure if this is supposed to be merged?

Reply:
It is, yes. Investigating https://github.com/cockroachdb/cockroach/issues/74919 separately.